ML-Frameworks/pytorch-aarch64/requirements.txt

```diff
@@ -27,15 +28,17 @@ PyYAML~=6.0.2
 regex==2024.9.11
 requests~=2.32.3
 safetensors~=0.4.5
+sentencepiece==0.2.0  # Torchchat
+setuptools>=78.1.1    # CVE-2024-6345 and CVE-2025-47273
 six~=1.16.0
 sympy~=1.13.1
 tiktoken~=0.9.0
 tokenizers~=0.21.0
 tqdm~=4.66.5
-transformers~=4.48.2
+transformers~=4.50.0  # CVE-2025-2099
 typing_extensions~=4.12.2
 tzdata==2024.2
 urllib3~=2.2.3
-sentencepiece==0.2.0  # Torchchat
 tomli==2.2.1          # Torchchat
```

Comment:
If torchchat example has been removed, do we still need this? If we still need it anyway, we should probably update or remove the comment.